projects/js-packages/publicize-components/src/hooks/use-media-restrictions/restrictions.js

```diff
@@ -117,10 +122,20 @@ export const RESTRICTIONS = {
 	},
 	[ 'instagram-business' ]: {
 		requiresMedia: true,
-		allowedMediaTypes: [ 'image/jpg', 'image/jpeg' ],
+		allowedMediaTypes: [ 'image/jpg', 'image/jpeg' ].concat( [ MP4, MOV, VIDEOPRESS ] ),
```

Comment:
We can probably concot this ourselves. 
```suggestion
		allowedMediaTypes: [ 'image/jpg', 'image/jpeg', MP4, MOV, VIDEOPRESS ],
```